### DISCUSSION TITLE
批量重命名的问题

### DISCUSSION THREAD

**Discussion by thinbug** [AlistGo/alist#4618] [Q&A]
批量重命名的正则如何写呢，又没范例？ 经常要改一下前缀S00Exx

**Comment by houzi5940** [comment]
原文件名：01.mp4，02.mp4，.....，n.mp4
正则：(\d+).mp4        #(注释：这个(\d+)的小括号很重要)
第二行：前缀S00E$1  #(注释：这个$1的1是正则中从左边数第1个小括号中的匹配值，$n 同理)
